resources/js/Pages/Layout.vue

```diff
@@ -144,12 +158,28 @@ export default defineComponent({
                 const languageSelectorRefs = this.$refs.languageSelector as any;
                 this.$nextTick(() => {
                     languageSelectorRefs.focus();
+                    this.changeLanguageSelectorMenuDirection();
                 });
             }
         },
         onClickOutsideLanguageSelector(): void {
             this.showLanguageSelector = false;
         },
+        changeLanguageSelectorMenuDirection(): void {
+            const mismatchFinderDropdownMenu = document.querySelector( '.mismatchfinder__language-selector');
+            const headerY = (document.querySelector( 'header') as HTMLElement).getBoundingClientRect().y;
```

Comment:
I think `.top` makes our intention clearer here (even though I'm a huge fan of succinct code)

Reply:
I do, `top` is easier to understand than `y`

Reply:
Done.